scripts/docker/software_router/log4j.properties

```diff
@@ -0,0 +1,16 @@
+# Root logger option
```

Comment:
100% sure I will forget to update this file. Why making a copy of the file which is in the repository root ?
Much more flexible and less error prone to copy the file in the Docker build context in the CI script.